addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -908,7 +908,7 @@
 		
 		"226"	//Battalion's Backup
 		{
-			"desp"			"Battalion's Backup: {positive}Spawns up to 3 Zombie Scouts on use"
+			"desp"			"Battalion's Backup: {positive}Spawns 3 Zombie Scouts on use"
```

Comment:
"up to" is still true when there not enough dead players or revival preference to respawn, but I'm not sure if that should be enough to keep it

Reply:
that's true, i forgot about that